<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>20</width>
      <height>721</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="ElevatorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>481</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QPushButton" name="e2_help">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>510</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>HELP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fire_button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>640</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>FIRE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>610</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_open">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OPEN</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_close">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>510</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_help">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>HELP</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="e1_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_obstacle">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOOR OBSTACLE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="e1_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elavator 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_open">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>510</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OPEN</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_f2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>450</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerout_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>640</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER OUTAGE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_close">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e2_obstacle">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>510</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOOR OBSTACLE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="e2_display">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="e2_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elavator 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="e1_f6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MAX WEIGHT = 200KG</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>580</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MAX WEIGHT = 200KG</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CURRENT WEIGHT = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e1_weight">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KG</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>580</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CURRENT WEIGHT = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>580</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KG</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e2_weight">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>580</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FloorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>331</width>
      <height>701</height>
     </rect>
    </property>
    <widget class="QPushButton" name="f6_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>640</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>580</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="f1_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QLabel" name="f5_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f2_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QLabel" name="f6_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>610</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 6</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="f3_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f2_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QLabel" name="f4_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 4</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="f6_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>640</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QLabel" name="f2_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f4_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f3_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f5_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QLabel" name="f1_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f1_up">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="f5_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>520</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QLabel" name="f3_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FLoor 3</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="f4_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
